<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reg</class>
 <widget class="QDialog" name="reg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="regButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>        输入账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>        输入密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd2_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>      再次输入密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
